common/v2/features/Dashboard/Dashboard.tsx

```diff
@@ -98,30 +98,6 @@ export default function Dashboard() {
                 </div>
               </div>
             </div>
-            <AddressMetadataContext.Consumer>
-              {({ readAddressMetadata }) => (
-                <TransactionContext.Consumer>
-                  {({ transactions }) => {
-                    return (
-                      <TransactionHistoryContext.Consumer>
-                        {({ transactionHistories }) => {
-                          return (
-                            <div className="Dashboard-desktop-bottom">
-                              <RecentTransactionList
-                                transactionHistories={transactionHistories}
-                                readAddressMetadata={readAddressMetadata}
-                                transactions={transactions}
-                                className="Dashboard-desktop-modifiedPanel"
```

Comment:
Please remove SCSS class

Reply:
This one is being used for `AccountList` still